packages/excubiae/contracts/package.json

```diff
@@ -25,6 +25,7 @@
         "access": "public"
     },
     "dependencies": {
+        "@ethereum-attestation-service/eas-contracts": "^1.7.1",
```

Comment:
I'm not sure I understand why there's two package.json files in this package and both are importing this dependency?

Reply:
Yes, we can safely remove them from the root package.